From 03e3451527852324ab5335d996c1c488434951c3 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Sun, 21 Feb 2021 15:36:25 +0100
Subject: [PATCH] Update Makefile

---
 quippy/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/quippy/Makefile b/quippy/Makefile
index 34f4481f8..31c565c02 100644
--- a/quippy/Makefile
+++ b/quippy/Makefile
@@ -74,7 +74,7 @@ F2PY_LINK_ARGS = $(shell ${PYTHON} -c 'import sys; print(" ".join([arg for arg i
 
 
 #################################
-.PHONY: all build install clean f90wrap
+.PHONY: all build install clean
 
 
 all: build
