From f58557c9cac6f3c3207347116ea09af35586b2f7 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Mon, 29 Nov 2021 15:35:02 -0700
Subject: [PATCH] Update setup.py

---
 quippy/setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/quippy/setup.py b/quippy/setup.py
index 93a369c61..08a216645 100644
--- a/quippy/setup.py
+++ b/quippy/setup.py
@@ -70,7 +70,7 @@ def build_extension(self, ext):
     packages=['quippy'],
     package_data={'quippy': package_data_files},
     cmdclass={'build_ext': my_build_ext },
-    ext_modules=[Extension('quippy._quippy', [])],
+    ext_modules=[Extension('quippy._quippy', extra_compile_args=["-fopenmp"], extra_link_args=["-fopenmp"])],
     entry_points={
         'console_scripts': ['gap_fit=quippy.cli:gap_fit',
                             'quip=quippy.cli:quip',
